src/test/resources/reasoning/existential_elim.vpr

```diff
@@ -0,0 +1,34 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+function eq(x: Int, y: Int): Bool {
+    x == y
+}
+
+method ex1()
+    requires exists x: Int :: { eq(x, 42) } eq(x, 42)
+{
+    obtain x:Int where eq(x, 42)
+    assert x == 42
+}
+
+
+field f: Int
+
+method ex2()
+{
+    //:: ExpectedOutput(typechecker.error)
+    obtain x: Ref where {x.f} acc(x.f)
+}
+
+function geq(x:Int, y:Int) : Bool
+{
+    x>=y
+}
+
+method ex3()
+{
+    assert geq(3, 0)
+    obtain x:Int, y:Int where {geq(x,y)} geq(x,y)
+    assert x>=y
```

Comment:
Method `ex2` should be moved to a separate testcase such that we can test that the type-checker correctly rejects `ex2` and that `ex3` verifies (modulo the `assert false`)
```suggestion
    assert x>=y
    //:: ExpectedOutput(assert.failed:assertion.false)
    assert false
```